docs/database/cosmosdb.md

```diff
@@ -440,5 +448,743 @@ It captures the number of active instances at any given time. Best practices dic
 <!-- END AUTOGENERATED TEXT -->
 <!-- endsemconv -->
 
+## Network Level Metrics
+
+### Metric: `db.client.cosmosdb.request.duration`
```

Comment:
would you report network-level metrics for both direct and gateway mode?

Reply:
yes..this metrics would be available for gateway and direct mode both

Reply:
why not rely on HTTP metrics for HTTP calls?

Reply:
We will loose the cosmos db specific dimensions and in HTTP we can't inject custom dimensions